pkgs/os-specific/darwin/apple-source-releases/bootstrap_cmds/default.nix

```diff
@@ -3,27 +3,20 @@
 appleDerivation {
   nativeBuildInputs = [ yacc flex ];
 
+  setSourceRoot = ''
+    sourceRoot="$pname-$version/migcom.tproj";
+  '';
```

Comment:
Are we doing this simply to remove the `cd migcom.tproj` line below and have `rm handler.c` in `postPatch` instead of `buildPhase`?

Reply:
We can merge postPatch to buildPhase, but `cd` or `sourceRoot` must have one. Would you like a heading `cd` in buildPhase?

Reply:
I think `cd` in `buildPhase` was just fine. It's not like we expect someone to distribute the contents of the "migcom.tproj" directory in a separate `src`.